templates/repo/blame.tmpl

```diff
@@ -52,7 +52,7 @@
 							{{if $.EscapeStatus.Escaped}}
 								<td class="lines-escape">
 									{{if $row.EscapeStatus.Escaped}}
-										<a href="" class="toggle-escape-button" title="{{template "repo/diff/escape_title" dict "diff" $row "locale" $.locale}}"></a>
+										<button class="toggle-escape-button button button-ghost" title="{{template "repo/diff/escape_title" dict "diff" $row "locale" $.locale}}"></button>
```

Comment:
Got a screenshot of this? https://github.com/go-gitea/gitea/pull/24812 will add padding to `button-ghost` and I think it won't be compatible, but I can revert there and add explicit padding.

Reply:
Escaped: ![image](https://github.com/go-gitea/gitea/assets/51889757/807d091f-0125-423d-acea-22402c29340d)
Unescaped: ![image](https://github.com/go-gitea/gitea/assets/51889757/ff1e9171-26ea-403c-9ecf-cd926ac0d663)

By the way, the reason I've argued with you and wxiaoguang that we need some sort of `button-ghost` class was exactly that I've planned this PR for quite some time :grin:.
So yeah, I'm torn on whether we should pad by default or not, as both cases occur within Gitea.
What we could do as well is add an additional `button-ghost-(un)padded` class or something like that.

Reply:
I think I'm fine with removing padding in https://github.com/go-gitea/gitea/pull/24812. Ideally we would have some size classes, but I would not use fomantic's naming, but [Chakra's 4 sizes](https://chakra-ui.com/docs/components/button/usage#button-sizes) plus one for zero padding, but that will be a more massive rewrite.

BTW, what am I looking for in those screenshots.

Reply:
Well… Since you commented on line 55, the :warning:s.

Reply:
I see. Those are too big for the line, btw. can we maybe use octicons that can be precisely sized instead of emojis there?

Reply:
Yes, but that is definitely worthy of its own PR as that needs to be changed in quite a few places then, and the complete logic needs to be rewritten there as I think we currently inject the `:warning:` through CSS.

Reply:
Removed the padding again in https://github.com/go-gitea/gitea/pull/24812.

Reply:
It does look a bit better for me, but the emoji button still stretches each line it's on by 2px, but I agree it's for another PR.

<img width="113" alt="Screenshot 2023-05-20 at 00 32 10" src="https://github.com/go-gitea/gitea/assets/115237/7da6ce11-d9e7-4420-af35-a0018bc70c4e">